.travis.yml

```diff
@@ -69,16 +61,3 @@ jobs:
         # Bootstrap the build harness, pull test image, and run unit tests.  
         - make component/test/unit
         - make security/scans
-        - make sonar/go  
-
-    - stage: release-ff
-      name: "Push commits to current release branch"
-      if: type = push AND branch =~ /^main$/
-      script:
-        - make release-ff
```

Comment:
Also delete the files `RELEASE_FF_BRANCH` and `RELEASE_MAIN_BRANCH` these were used by release-ff and not needed anymore.

Reply:
Good catch! I just removed those files.